PXC-3643: [MTR] Tests fail with keyring warnings in error log

venkatesh-prasad-v opened the pull request:
https://jira.percona.com/browse/PXC-3643

Background:
After PXC-3092, we log a warning to notify customer when keyring
is enabled but pxc_encrypt_cluster_traffic is disabled.

Fix: Add the keyring warning to the global suppression list.

altmannmarcelo reviewed (changes requested):
Patch looks good. I would only request to remove the suppression on tests from https://github.com/percona/percona-xtradb-cluster/pull/1213/files since they are not necessary on individual tests anymore.

Commit:
PXC-3643: [MTR] Tests fail with keyring warnings in error log

https://jira.percona.com/browse/PXC-3643

Background:
After PXC-3092, we log a warning to notify customer when keyring
is enabled but pxc_encrypt_cluster_traffic is disabled.

Fix: Add the keyring warning to the global suppression list.

altmannmarcelo reviewed (approved):
LGTM
